app/helpers/tel-format.js

```diff
@@ -1,13 +1,93 @@
-import { helper } from '@ember/component/helper';
+const NR_WITH_COUNTRY = /^(?:\+|00)(\d\d)(\d{8,9})$/;
+const NR_WITHOUT_COUNTY = /^0(\d{8,9})$/;
+const SHORT_NR = /^\d{4}$/;
+const FREE_NR = /^0800(\d{4,9})$/;
+const TEL_PREFIX = /^tel\:/;
 
-export default helper(function telFormat([tel]) {
-  if (!tel) throw new Error('Tel format helper requires one parameter');
+/**
+ *
+ * @param {string} digits
+ */
```

Comment:
**Nitpick:** Since you already added a placeholder for documentation, could you add some explanation what the functions do? This might save a future developer from having to parse the code to understand what the different functions do.